content/en/docs/tutorials/configuration/pod-sidecar-containers.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Adopting Sidecar Containers
+content_type: tutorial
+weight: 40
+min-kubernetes-server-version: 1.29
+---
+
+<!-- overview -->
+
+This section is relevant for people adopting [sidecar containers](/docs/concepts/workloads/pods/sidecar-containers/) for their workloads.
+
```

Comment:
We should (I think need to) explain that you can use sidecars with older clusters. Otherwise, we risk seriously misleading readers.

Reply:
added a paragraph